aireview/poetry.lock

```diff
@@ -1,5 +1,188 @@
 # This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
 
```

Comment:
**style**: Lock file modifications should be committed separately from code changes to make dependency updates more traceable and reviewable.